app/src/main/java/net/gsantner/markor/ui/SearchOrCustomTextDialogCreator.java

```diff
@@ -640,6 +645,54 @@ public static void showSetPasswordDialog(final Activity activity) {
         }
     }
 
+    private static String getFormatSnippetExtension(@StringRes final int format) {
+        switch (format) {
+            case (TextFormat.FORMAT_MARKDOWN):
+                return ".md";
+            case (TextFormat.FORMAT_PLAIN):
+                return ".txt";
+            case (TextFormat.FORMAT_TODOTXT):
+                return ".todo";
+            case (TextFormat.FORMAT_ZIMWIKI):
+                return ".zim";
+        }
+        return "";
```

Comment:
Each file format has it's own snippet extension

Reply:
I'd keep it simple, just text files which content can be inserted. no format/filetype specifics

specifically also - html for example has no specific markor format assigned yet. But user might want to insert html block independent to what the current file is written in.